README.md

```diff
@@ -14,49 +14,54 @@ Add [`@PullPartyBot`](https://t.me/PullPartyBot) to the chat to be able to creat
 > all commands are not case-sensitive to support RUDE mode \
 > usernames are char sequences of letters, digits and underscores of 5-32 length
 
-+ `/create partyName user1 user2...` - create party with mentioned users
++ `/create partyName user1 user2...` - create new party with mentioned users
 
-    > You can enter users with or without `@` symbol \
-    You can't create empty party \
-    `@admins` party is reserved and already created \
-    `@` prefix will be removed from party name \
-    `@` symbol is not allowed in party name \
-    You can't add single user to one group more than once
+    > Users within the party are not repeating \
+    Party should consist of at least one user \
+    You can enter users with or without `@` symbol \
+    `@admins` is a reserved party and already exists \
+    `@`, `!`, `,`, `,`, `?`, `:`, `;`, `(`, `)` symbols and trailing `-` are not allowed in the party name
 
-+ `/change partyName user2 user3...` - change an existing party
++ `/change partyName user1 user2...` - change an existing party
 
     > Follows all `/create` method's rules \
     You can't change `@admins` party
 
 
-+ `/delete partyName1 partyName2...` - forget the parties like it never happened
++ `/delete partyName1 partyName2...` - delete the parties you provided
 
-    > Availible only to admins \
-    You can't delete `@admins` party
+    > Only admins have access to `/delete` \
```

Comment:
By the way, wouldn't it be useful to make an `admins` button attached to such messages (__sorry, but this command type is only available for admins__). This button will tag admins, basically. Positive sides for users: they won't have to write `@admins` or `/party admins` by themselves.

Or probably that's overthinking

Reply:
`Or probably that's overthinking`
^^^^^^^^^^^^^^^^^^^^^
\ \ \ \ \ \ \ \ \\ this / / / / / / / / / / /